src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -146,7 +148,7 @@
         var allFieldsBySObject = this.groupFieldsBySObject(everyField);
 
         const opportunitySObjectName = "Opportunity";
```

Comment:
Could/should we get these from our labels file? Also, can you change these both to single quotes?